src/lib.rs

```diff
@@ -89,6 +89,24 @@ pub struct PtObjects {
     vehicle_journeys_to_stop_points: ManyToMany<VehicleJourney, StopPoint>,
     transfers_to_stop_points: ManyToMany<Transfer, StopPoint>,
 
+    // For comments, we don't want to use it like a shortcut.
+    // This is the reason why the weight is equal to 100 (for short path algorithm).
+    // example :
+    // stop area --------> stop point --------> route
+    //             w = 1               w = 1.9        total weight = 2.9 (best)
+    // stop area -------->   comment  --------> route
+    //            w = 100              w = 200        total weight = 200
+    #[get_corresponding(weight = "100")]
+    lines_to_comments: ManyToMany<Line, Comment>,
+    #[get_corresponding(weight = "100")]
+    routes_to_comments: ManyToMany<Route, Comment>,
+    #[get_corresponding(weight = "100")]
+    vehicle_journeys_to_comments: ManyToMany<VehicleJourney, Comment>,
+    #[get_corresponding(weight = "100")]
+    stop_areas_to_comments: ManyToMany<StopArea, Comment>,
+    #[get_corresponding(weight = "100")]
+    stop_points_to_comments: ManyToMany<StopPoint, Comment>,
```

Comment:
This may not be really interesting as it will only be used to convert `Vec<Idx<Comment>>` to `IdxSet<Comment>`.

But we will need something like that on `Collection<T>` (not sure of the name, @datanel any idea?):
```rust
fn iter_from<I>(&self, iter: I) -> impl Iterator<Item = &Comment>
where
    I: IntoIterator,
    I::Item: Into<Idx<Comment>>,
```
for doing something like that:
```rust
for comment in pt_objects.comments.iter_from(&stop_area.comment_links) {
    println!("{}", comment.name);
}
```
And `comment_links` may be a bad name, not representing what it is. Maybe `comments` would be better? @datanel what do you think?